iOS/REFrostedViewController/License.md

```diff
@@ -0,0 +1,17 @@
+**Xamarin is not responsible for, nor does it grant any licenses to, third-party packages. Some packages may require or install dependencies which are governed by additional licenses.**
+
+Note: This component depends on [REFrostedViewController](https://github.com/romaonthego/REFrostedViewController), which is subject to the [MIT License](https://github.com/romaonthego/REFrostedViewController/blob/master/LICENSE)
+
+### Xamarin Component for REFrostedViewController
```

Comment:
The text above this line is not needed unless you are using Xamarin Build Download.  

If you use Xamarin Build Download you are not redistributing the third party content and you use the lines above.  If you are not using Xamarin Build Download then you are redistributing the third party content and you use the External-Dependency-Info.txt/THIRD-PARTY-NOTICES.txt file to give notice of the additional license info.